### ISSUE TITLE
出现腾讯云安全ID导致无法通过从本地传到github

### ISSUE THREAD

**Issue by turingzhi** [blinkfox/hexo-theme-matery#923]
—— Tencent Cloud Secret ID ———————————————————————————
remote:        locations:
remote:          - commit: 525a9ba97031c6b67fb0e38d5dacfec37614faf4
remote:            path: libs/twikoo/twikoo.all.min.js:2

**Comment by zlH518** [comment]
Has this issue been resolved?

**Comment by zlH518** [comment]
> —— Tencent Cloud Secret ID ——————————————————————————— remote: locations: remote: - commit: 525a9ba97031c6b67fb0e38d5dacfec37614faf4 remote: path: libs/twikoo/twikoo.all.min.js:2

maybe this url can help you --> https://blog.51cto.com/u_12763213/10979913#:~:text=%E8%A7%A3%E5%86%B3%E6%96%B9%E6%B3%95%EF%BC%8C%E8%BF%99%E9%87%8C%E6%9C%89%E4%B8%89%E7%A7%8D%E6%96%B9%E6%B3%95%EF%BC%9A%201%EF%BC%89%E5%88%A0%E9%99%A4%E4%BB%A3%E7%A0%81%E9%87%8C%E9%9D%A2%E7%9A%84%E5%AE%9A%E4%B9%89%E7%9A%84%20secret%20%E7%9B%B8%E5%85%B3%E7%9A%84%E5%80%BC%EF%BC%8C%E9%87%8D%E6%96%B0commit%EF%BC%8C%E5%86%8Dpush%E3%80%82%202%EF%BC%89%E4%BF%AE%E6%94%B9%E4%BB%93%E5%BA%93%E7%9A%84%E9%85%8D%E7%BD%AE%E5%85%B3%E9%97%ADsecret%E6%A3%80%E6%B5%8B%EF%BC%8C%E6%AD%A5%E9%AA%A4%E5%A6%82%E4%B8%8B%EF%BC%9A,3%EF%BC%89%E9%80%9A%E8%BF%87%20push%20%E6%8F%90%E7%A4%BA%E7%BB%99%E5%87%BA%E7%9A%84%E8%AE%BF%E9%97%AE%E9%93%BE%E6%8E%A5%E8%BF%9B%E8%A1%8C%E8%AE%B8%E5%8F%AF%E6%B7%BB%E5%8A%A0%EF%BC%8C%E6%AD%A5%E9%AA%A4%E5%A6%82%E4%B8%8B%EF%BC%9A%20%E6%9C%80%E5%90%8E%EF%BC%8C%E5%86%8D%E6%AC%A1%20push%20%E5%B0%B1%E5%8F%AF%E4%BB%A5%E4%BA%86%E3%80%82

**Comment by turingzhi** [comment]
Yes. I have solved it. Sorry that I forgot to update this issue.
